docs/sections/user_guide/cli/index.rst

```diff
@@ -7,4 +7,5 @@ CLI
    mode_config
    mode_fv3
    mode_rocoto
+   mode_sfc_climo_gen
```

Comment:
How would you feel about breaking this out into sections for Tools and Drivers?

Reply:
I'd feel good about that. Done.